survey-runner.tf

```diff
@@ -174,7 +221,37 @@ module "survey-runner-on-ecs" {
               "dynamodb:GetItem"
           ],
           "Resource": "${module.survey-runner-dynamodb.submitted_responses_table_arn}"
+      },
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "dynamodb:PutItem",
+              "dynamodb:GetItem",
+              "dynamodb:DeleteItem"
+          ],
+          "Resource": "${module.survey-runner-dynamodb.questionnaire_state_table_arn}"
+      },
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "dynamodb:PutItem",
+              "dynamodb:GetItem",
+              "dynamodb:DeleteItem"
+          ],
+          "Resource": "${module.survey-runner-dynamodb.eq_session_table_arn}"
+      },
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "dynamodb:PutItem",
+              "dynamodb:GetItem"
```

Comment:
Technically this doesn't need `dynamodb:GetItem` but not sure if it will work without?

Reply:
I'd be surprised if the `ConditionExpression` we use when putting works (or should work) without `GetItem`, going to leave it in to be safe.

Reply:
i would wip it out ( ;) ) and give it a try?

Reply:
works without `GetItem` 😮